glean-core/metrics.yaml

```diff
@@ -596,3 +596,27 @@ glean.validation:
       - glean-team@mozilla.com
     expires:
       never
+  pings_submitted:
+    type: labeled_counter
+    description: >
```

Comment:
Always getting confused by all the various YAML multi-line string options.
I think `>` will remove newlines (unless it's double lines, which turn into a single newline).
`|` keeps newlines as written. Is that what we should use here to keep the formatting?

Reply:
Yeah, we should use `|` throughout to avoid surprises.